cmd/clusterctl/test/e2e/go.mod

```diff
@@ -6,13 +6,13 @@ require (
 	github.com/onsi/ginkgo v1.11.0
 	github.com/onsi/gomega v1.8.1
 	github.com/pkg/errors v0.9.0
-	k8s.io/api v0.0.0-20191121015604-11707872ac1c
```

Comment:
@fabriziopandini @Arvinderpal 

This is breaking the build so I am adding it here.